client/extensions/woocommerce/state/sites/setup-choices/actions.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Internal dependencies
+ */
+import {
+	areSetupChoicesLoaded,
+	areSetupChoicesLoading
+} from './selectors';
+import { setError } from '../status/wc-api/actions';
+import {
+	WOOCOMMERCE_SETUP_CHOICES_REQUEST,
+	WOOCOMMERCE_SETUP_CHOICES_REQUEST_SUCCESS
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc